pandasai/__init__.py

```diff
@@ -172,6 +193,11 @@ def run_code(
             return result
         except:
             return captured_output
+        
+    def create_new_cell(self, contents: str):
```

Comment:
Maybe we can move it to a dedicate helper (like notebook or anything like that?

In this way we keep the code cleaner.